rllib/__init__.py

```diff
@@ -58,6 +61,7 @@ def setup(self, config):
             register_trainable(alias, _see_contrib(alias))
 
 
+_register_all()
```

Comment:
looks a bit suspicious. why didn't we have to register anything before this PR?

Reply:
We have explicitly called `_register_all()` in places where it was needed. for example in the tune_restore_test() we explicitly call this in the `tear_down()` function to reimport all the classes after the tune actor dies. For some reason that does not work for the connectors. We could have also called register_all when importing `algorithm.py` but I don't think that's the right place to call register_all. Basically the logic is that as soon as you import rllib all the namespaces with rllib should be recognizable globally.